src/Analysis/Codelyzer.Analysis.CSharp/Handlers/InterfaceDeclarationHandler.cs

```diff
@@ -18,24 +17,17 @@ public class InterfaceDeclarationHandler : UstNodeHandler
             var interfaceSymbol = SemanticHelper.GetDeclaredSymbol(syntaxNode, SemanticModel, OriginalSemanticModel);
             InterfaceDeclaration.Identifier = syntaxNode.Identifier.ToString();
 
-
             if (interfaceSymbol != null)
             {
-                if (interfaceSymbol.BaseType != null)
-                {
-                    InterfaceDeclaration.BaseType = interfaceSymbol.BaseType.ToString();
-                    InterfaceDeclaration.BaseTypeOriginalDefinition = GetBaseTypOriginalDefinition(interfaceSymbol);
-                }
-
+                InterfaceDeclaration.FullIdentifier = GetFullIdentifier(interfaceSymbol);
                 InterfaceDeclaration.Reference.Namespace = GetNamespace(interfaceSymbol);
                 InterfaceDeclaration.Reference.Assembly = GetAssembly(interfaceSymbol);
                 InterfaceDeclaration.Reference.Version = GetAssemblyVersion(interfaceSymbol);
                 InterfaceDeclaration.Reference.AssemblySymbol = interfaceSymbol.ContainingAssembly;
-                InterfaceDeclaration.FullIdentifier = string.Concat(InterfaceDeclaration.Reference.Namespace, ".", InterfaceDeclaration.Identifier);
 
-                if (interfaceSymbol.Interfaces != null)
+                if (interfaceSymbol.AllInterfaces != null)
                 {
-                    InterfaceDeclaration.BaseList = interfaceSymbol.Interfaces.Select(x => x.ToString())?.ToList();
+                    InterfaceDeclaration.BaseList = interfaceSymbol.AllInterfaces.Select(x => x.ToString())?.ToList();
```

Comment:
@marknfawaz From my testing, the `BaseList` contains the base interfaces without the namespace. See the screenshot below. I would like to include the namespace to have a full identifier.  Do you know how to add the namespace?
<img width="516" alt="IObjectType" src="https://user-images.githubusercontent.com/108032883/177895995-02358dd1-f331-4adb-9c22-fa53899deb62.png">

Reply:
Let's discuss. The baselist is a list we are populating, so most likely we should be able to update.

Reply:
As discussed offline, we keep it as is. Users will need to look for the using statements to determine the namespace if the interface or class is from System or System.* space.